AUTHORS

```diff
@@ -10,3 +10,4 @@
 Google LLC
 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V. for its Fraunhofer Institute for Material Flow and Logistics (IML)
 Nod, Inc.
+Advanced Micro Devices
```

Comment:
```suggestion
Advanced Micro Devices, Inc.
```

?

Reply:
Would be good to confirm, yeah. Looks like some recent files in https://github.com/nod-ai/SHARK-Turbine use `, Inc.` in copyright comments but I've been using without that in https://github.com/nod-ai/SHARK-TestSuite